### PR TITLE
Issue 558 change plotting labels

### PR DESCRIPTION
**Pull request by achevali**

### CODE REVIEW

File: src/main/resources/org/broadinstitute/hellbender/utils/plotter/ACNVResultsPlotting.R [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #NOTE: the Java wrapper for this script first sources CNV_plotting_library.R
-options(error = quote({dump.frames(dumpto = "plotting_dump", to.file = TRUE); q()}))    # Useful for debugging
+options(error = quote({dump.frames(dumpto = "plotting_dump", to.file = TRUE); q(status = 1)}))    # Useful for debugging
```
Comment (LeeTL1220) [inline src/main/resources/org/broadinstitute/hellbender/utils/plotter/ACNVResultsPlotting.R]: Excellent!  Now we should get a proper non-zero return code if plotting fails.

File: src/main/resources/org/broadinstitute/hellbender/utils/plotter/ACNVResultsPlotting.R [hunk 2]
```diff
@@ -32,7 +32,9 @@ create_acnv_plots_file = function(sample_name, snp_counts_file, coverage_file, s
 	segments = read.table(segments_file, sep="\t", stringsAsFactors=FALSE, header=TRUE, check.names=FALSE)
     snp_counts[,"CONTIG"] = convert_XY_to_23_24(snp_counts[,"CONTIG"])
     coverage = read.table(coverage_file, sep="\t", stringsAsFactors=FALSE, header=TRUE, check.names=FALSE)
-    names(coverage)[names(coverage) == sample_name] = "VALUE"
+
```
Comment (LeeTL1220) [inline src/main/resources/org/broadinstitute/hellbender/utils/plotter/ACNVResultsPlotting.R]: Does this mean that you will take the fourth column as the sample name?  If so, that is problematic...

**Comment by LeeTL1220** [comment]
@achevali Please see one comment.

File: src/main/resources/org/broadinstitute/hellbender/utils/plotter/ACNVResultsPlotting.R (hunk 2 above)
Comment (achevali) [inline src/main/resources/org/broadinstitute/hellbender/utils/plotter/ACNVResultsPlotting.R]: It takes the first 4 columns as is, then changes the 5th to the sample_name.

**Comment by achevali** [comment]
Back to you

File: src/main/resources/org/broadinstitute/hellbender/utils/plotter/ACNVResultsPlotting.R (hunk 2 above)
Comment (LeeTL1220) [inline src/main/resources/org/broadinstitute/hellbender/utils/plotter/ACNVResultsPlotting.R]: Does that work in all cases?  I thought you were going to do a set difference on the column names.

Let's discuss in person.

**Comment by LeeTL1220** [comment]
@achevali Let's just discuss in person.  What you have now may be fine.

**Comment by achevali** [comment]
Firehose failures will be fixed by code change already in place.

**Comment by LeeTL1220** [comment]
@achevali After seeing the plots you showed me, merge when ready!

**Pull request merged**